commands/operator-sdk/cmd/test.go

```diff
@@ -49,20 +49,21 @@ func NewTestCmd() *cobra.Command {
 	testCmd.Flags().StringVarP(&crdManifestPath, "crd", "c", "deploy/crd.yaml", "Path to CRD manifest")
```

Comment:
go test also takes in `-c` and `-o` flags. Will that conflicts to the operator-sdk test flags `-c` and `-o`?

Reply:
That won't cause conflict. The way that `Flags().SetInterspersed(false)` works is that any arguments/flags given after the first argument it sees after flags (argument being anything that doesn't start with a `-`) is not processed and instead is given to us in `args []string`. For instance, if I run `operator-sdk test -t ./test/e2e -k ~/.kube/config args -v -parallel 2`, I get this as my `args []string`: `["args" "-v" "-parallel" "2"]`.